FirebaseAdmin/FirebaseAdmin.Tests/Auth/Providers/OidcProviderConfigTest.cs

```diff
@@ -754,6 +785,16 @@ public IEnumerator<object[]> MakeEnumerator()
                     },
                     "Malformed issuer string: not a url",
                 };
+                yield return new object[]
```

Comment:
I think another case we might want to prevent is trying to set both response types to `false`.

Reply:
Done